AK/Tests/TestTime.cpp

```diff
@@ -238,6 +238,11 @@ TEST_CASE(truncation)
     EXPECT_EQ(TIME(-2, -800'800'800).to_truncated_milliseconds(), -2'800);
     EXPECT_EQ(TIME(-2, -800'800'800).to_truncated_microseconds(), -2'800'800);
 
+    EXPECT_EQ(TIME(-9223372036854776, 193'000'000).to_truncated_milliseconds(), -0x7fff'ffff'ffff'ffff);
+    //EXPECT_EQ(TIME(-9223372036854776, 193'000'000).to_truncated_microseconds(), -0x7fff'ffff'ffff);
+    EXPECT_EQ(TIME(9223372036854776, 193'000'000).to_truncated_milliseconds(), 0x7fff'ffff'ffff'ffff);
```

Comment:
It's rather uninteresting to check for the return value `0x7fff'ffff'ffff'ffff`, since any bug that causes one overflow too many will trigger this return value easily.

Let's also test this:
```
EXPECT_EQ(TIME(9223372036854776, 192'000'000).to_truncated_milliseconds(), 0x7fff'ffff'ffff'fffe);
EXPECT_EQ(TIME(9223372036854776, 193'000'000).to_truncated_milliseconds(), 0x7fff'ffff'ffff'ffff);
EXPECT_EQ(TIME(9223372036854776, 194'000'000).to_truncated_milliseconds(), 0x7fff'ffff'ffff'ffff);
```

… and now you know why many tests here occur in triplets :)
(One before the overflow, one that causes the extremal value as the correct result anyway, and one that overflows and causes the extremal value by saturation.)